android-attestation/src/main/kotlin/AndroidAttestationConfiguration.kt

```diff
@@ -93,7 +93,13 @@ class AndroidAttestationConfiguration @JvmOverloads constructor(
     val trustAnchors: List<PublicKey> = listOf(
         KeyFactory.getInstance("RSA")
             .generatePublic(X509EncodedKeySpec(Base64.getDecoder().decode(GOOGLE_ROOT_CA_PUB_KEY)))
-    )
+    ),
+
+    /**
+     *  Tolerance in seconds added to verification date 
+     *
+     */
+    val verificationTimeOffset: Int = 0
```

Comment:
I'd just rename the property to `verificationSecondsOffset` or something similar

Reply:
done